guides/common/modules/con_managing-content-views.adoc

```diff
@@ -34,10 +34,11 @@ The general workflow for creating Content Views for filtering and creating snaps
 
 . Create a Content View.
 . Add one or more repositories that you want to the Content View.
-. Optionally, create one or more filters to refine the content of the Content View.
-. Optionally, resolve any package dependencies for a Content View.
+. Optional:
+.. Create one or more filters to refine the content of the Content View.
+.. Resolve any package dependencies for a Content View.
```

Comment:
The feedback from https://bugzilla.redhat.com/show_bug.cgi?id=2123123 suggest we make these steps into substeps.
```suggestion
.. Optional: Create one or more filters to refine the content of the Content View.
.. Optional: Resolve any package dependencies for a Content View.
```

Would something like this be feasible?

Reply:
I don't think that we can mark those as sub-steps of the previous step, because previous step describes a separate action.